io.github.mightycreak.Diffuse.yml

```diff
@@ -0,0 +1,19 @@
+app-id: io.github.mightycreak.Diffuse
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: diffuse
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=home
```

Comment:
Ideally, you should switch to https://developer.gnome.org/gtk3/stable/gtk3-GtkFileChooserNative.html in the future – that will allow you to drop this line as it will be handled by the portal.

Reply:
Thank you! I created an issue in my repo: https://github.com/MightyCreak/diffuse/issues/78